src/dbSta/src/dbReadVerilog.cc

```diff
@@ -280,14 +345,83 @@ void Verilog2db::makeDbModule(Instance* inst, dbModule* parent)
                     network_->name(cell));
       return;
     }
+    if (hierarchy_) {
+      // make the module bterms
+      CellPortIterator* cp_iter = network_->portIterator(cell);
+      while (cp_iter->hasNext()) {
+        Port* port = cp_iter->next();
+        /* Ports are prefixed by instance name*/
+        if (network_->isBus(port)) {
+          const char* port_name = network_->name(port);
+          const char* cell_name = network_->name(cell);
+          int from = network_->fromIndex(port);
+          int to = network_->toIndex(port);
+          string key = "bus_msb_first ";
+          key += key + port_name + " " + cell_name;
+          key += port_name;
+          odb::dbBoolProperty::create(block_, key.c_str(), from > to);
+          // Make a modbterm for each bus bit
+          int start_index = from < to ? from : to;
+          int end_index = from < to ? to : from;
+          for (int i = start_index; i <= end_index; i++) {
+            // use actual here
+            std::string bus_bit_port = port_name + std::string("[")
+                                       + std::to_string(i) + std::string("]");
+            dbModBTerm* bmodterm
+                = dbModBTerm::create(module, bus_bit_port.c_str());
+            dbIoType io_type = staToDb(network_->direction(port));
+            bmodterm->setIoType(io_type);
+          }
+        } else {
+          std::string port_name = network_->name(port);
+          dbModBTerm* bmodterm = dbModBTerm::create(module, port_name.c_str());
+          dbIoType io_type = staToDb(network_->direction(port));
+          bmodterm->setIoType(io_type);
+          debugPrint(logger_,
+                     utl::ODB,
+                     "dbReadVerilog",
+                     1,
+                     "Created module bterm {} ",
+                     bmodterm->getName());
+        }
+      }
+      // make the instance iterms
+      InstancePinIterator* ip_iter = network_->pinIterator(inst);
+      while (ip_iter->hasNext()) {
+        Pin* cur_pin = ip_iter->next();
+        std::string pin_name_string(network_->name(cur_pin));
+        pin_name_string = makePinName(pin_name_string);
```

Comment:
Perhaps using `portName` instead of `name` here would remove the need for makePinName?